cli/main.vala

```diff
@@ -69,6 +85,8 @@ public class LoginDHelper.Application : GLib.Application {
             Bus.unown_name (own_id);
         }
 
+        reload_logind ();
```

Comment:
There is no need for a dedicated function here, as the function only has 4 lines, but adding a comment with something like "We need to restart systemd-logind to ensure that the lid settings are taken into account" might help to understand why this is required